website/docs/indexes.md

```diff
@@ -45,22 +45,40 @@ If it's `-1`, it specifies a descending order direction for the index.
 ### Compound Indexes
 
 For compound indexes, you can create an index key combining multiple fields together as a key.
-Below is an example of a compound index that uses `price` and `category` fields from the `products` collection as the index key.
+Below is an example of a compound index that uses `price` and `category` fields
+from the `products` collection as the index key:
 
 ```js
 db.products.createIndex({ price: 1, category: 1 })
 ```
 
-:::note
+### Unique Indexes
+
+You can create unique indexes to ensure that the indexed fields do not contain duplicate values.
+To create a unique index, set the `unique` option as `true` when calling `createIndexes()` command.
+
+Below is an example of a unique index for the `name` field from the `products` collection:
+
+```js
+db.products.createIndex({ name: 1 }, { unique: true })
+```
+
+Unique indexes can be compound.
+Here is an example of a unique index consisting
+of the `category` and `name` fields from the `products` collection:
+
+```js
+db.products.createIndex({ category: 1, name: 1 }, { unique: true })
+```
+
+### Index creation details
```

Comment:
@rumyantseva A separate section may be quite confusing since we have one on `how to create indexes`. Let's add this to the section above on `how to create indexes`.

Reply:
It can be a subsection there too or just as extra details after this sentence `You can create single field indexes or compound indexes.`

Reply:
"Index creation details" is a subsection of "How to create indexes", not a top-level section

Reply:
@Fashander Please send a new PR to improve that page if needed